Autocomplete/predefined-items-and-recent-searches/src/index.js

```diff
@@ -0,0 +1,63 @@
+import algoliasearch from 'algoliasearch/lite';
+import {
+  autocomplete,
+  getAlgoliaHits,
+  reverseHighlightItem,
+} from '@algolia/autocomplete-js';
+import { predefinedItems } from './predefined-items';
+import { recentSearches } from './recent-searches';
+
+const searchClient = algoliasearch(
+  'latency',
+  '6be0576ff61c053d5f9a3225e2a90f76'
+);
+
+autocomplete({
+  container: '#autocomplete',
+  plugins: [predefinedItems, recentSearches],
+  openOnFocus: true,
+  onStateChange({ state, prevState }) {
```

Comment:
Is that supposed to be here?

Reply:
It's to demonstrate for users a way to get notified of query change. This example isn't a standalone, but they need to do something else with it.